open-build-service/system-users-qmail.conf

```diff
@@ -0,0 +1,17 @@
+# Type Name ID GECOS [HOME]
```

Comment:
This is used only by OpenSUSE? Comment plz

Reply:
Comment has been updated as following. AFAIK, the sysusers_generate_pre macro works only on Fedora and openSUSE > 15 versions. I will test this as part of the next release. If this works for all distributions the useradd commands can be removed from the preinst scripts

```
# Used in notqmail.spec for user creation
# in openSUSE systems
# The sysusers_generate_pre macros generates
# notqmail.pre from system-users-qmail.conf
# This works on openSUSE_LEAP versions >= 15.0 and SLE >= 15
# %if 0%{?suse_version} >= 1500 || 0%{?sles_version} >= 15
```